package.json

```diff
@@ -2,35 +2,36 @@
   "private": true,
   "scripts": {
     "lint": "vue-cli-service lint --max-warnings 0 --no-fix .",
-    "start": "nuxt dev --no-clear",
-    "generate": "nuxt generate",
+    "dev": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
```

Comment:
Preview the build static files locally - this mimics our production environment and is more performant than `yarn dev` (with hot module replacement)
https://docs.astro.build/en/reference/cli-reference/#astro-preview